File: PracticasProgramacion/Practica_26/python/main.py
"""
Practica_26 - Escribir y leer TXT (datos basicos, arreglos, autos, personas ADT).
"""

from manejador_txt import ManejadorTXT

if __name__ == "__main__":
    man = ManejadorTXT()
    man.escribir_datos_basicos("datos_basicos.txt")
    man.escribir_arreglo("arreglo.txt")
    man.escribir_autos("autos.txt")
    man.escribir_personas("personas.txt")

    print("Leyendo datos basicos:")
    man.leer_y_mostrar("datos_basicos.txt")
    print("\nLeyendo arreglo:")
    man.leer_y_mostrar("arreglo.txt")
    print("\nLeyendo autos:")
    man.leer_y_mostrar("autos.txt")
    print("\nLeyendo personas (ADT):")
    man.leer_y_mostrar("personas.txt")

File: PracticasProgramacion/Practica_26/cpp/main.cpp
/**
 * Practica_26 - Escribir y leer TXT (datos basicos, arreglos, creados, ADT)
 */

#include "ManejadorTXT.h"
#include <iostream>

int main() {
    ManejadorTXT man;

    man.escribirDatosBasicos("datos_basicos.txt");
    man.escribirArreglo("arreglo.txt");
    man.escribirAutos("autos.txt");
    man.escribirPersonas("personas.txt");

    std::cout << "Leyendo datos basicos:\n";
    man.leerYMostrar("datos_basicos.txt");
    std::cout << "\nLeyendo arreglo:\n";
    man.leerYMostrar("arreglo.txt");
    std::cout << "\nLeyendo autos:\n";
    man.leerYMostrar("autos.txt");
    std::cout << "\nLeyendo personas (ADT):\n";
    man.leerYMostrar("personas.txt");

    return 0;
}
